### PR TITLE
Release version 1.13 (#249)

### PR DESCRIPTION
**Pull request by whimboo**
This is the PR necessary for issue #249. Just a version bump and changes to the history file.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 
 import os
-from setuptools import setup, find_packages
```
Comment (Nebelhom) [inline setup.py]: Why are we removing find_packages?

**Comment by Nebelhom** [comment]
@whimboo 

I cannot see much wrong with it. Just answer the remark satisfactorily and you will have a +1 from me.

File: setup.py (hunk 1 above)
Comment (whimboo) [inline setup.py]: It was never used in this file, so just an orphaned include.

**Comment by whimboo** [comment]
This PR has been merged as https://github.com/mozilla/mozdownload/commit/ea791a65f2900a3923e01b46b8cc88fabd6d5838

**Pull request closed without merging**